File: app/src/main/java/org/reloaded/updater/api/ApiInterface.java
package org.reloaded.updater.api;

import retrofit2.Call;
import retrofit2.http.Field;
import retrofit2.http.FormUrlEncoded;
import retrofit2.http.POST;

public interface ApiInterface {
    @FormUrlEncoded
    @POST("checkupdates.php")
    Call<Response> checkupdates(@Field("device") String device, @Field("device_id") String device_id, @Field("build_date") String build_date);
}

File: app/src/main/java/org/reloaded/updater/api/ApiInterface.kt
package org.reloaded.updater.api

import retrofit2.Call
import retrofit2.http.Field
import retrofit2.http.FormUrlEncoded
import retrofit2.http.POST

interface ApiInterface {
    @FormUrlEncoded
    @POST("check_update")
    fun checkupdates(@Field("device") device: String, @Field("device_id") device_id: String, @Field("version") version: String): Call<Response>
}
